ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/RandomnessProvider.scala

```diff
@@ -5,19 +5,23 @@ package com.daml.ledger.api.benchtool.submission
 
 trait RandomnessProvider {
   def randomDouble(): Double // 0.0 <= randomDouble() < 1.0
-  def randomBytes(n: Int): Array[Byte]
+  /** Guarantees that each character will take exactly one byte in UTF-8.
+    */
+  def randomAsciiString(n: Int): String
   def randomNatural(n: Int): Int // 0 <= randomNatural(n) < n
 }
 
 object RandomnessProvider {
   object Default extends RandomnessProvider {
     private val r = new scala.util.Random(System.currentTimeMillis())
     override def randomDouble(): Double = r.nextDouble()
-    override def randomBytes(n: Int): Array[Byte] = {
-      val arr = Array.ofDim[Byte](n)
-      r.nextBytes(arr)
-      arr
-    }
     override def randomNatural(n: Int): Int = r.nextInt(n)
+    override def randomAsciiString(n: Int): String = {
```

Comment:
Did you consider just using `scala.util.Random.nextString(n)` ? I am not sure what benefit does it bring in a testing library to have these wrappers.

Reply:
The problem with this is that you can get characters that need more then one byte and so it's hard to related the requested size payload size in bytes to the length of this random string in terms of number of characters